packages/apollo-server-caching/package.json

```diff
@@ -23,11 +23,16 @@
     "node": ">=8"
   },
   "dependencies": {
-    "http-cache-semantics": "^4.0.0"
+    "http-cache-semantics": "^4.0.0",
+    "memcached": "^2.2.2",
+    "redis": "^2.8.0"
   },
   "devDependencies": {
     "@types/jest": "^23.0.0",
+    "@types/memcached": "^2.2.5",
+    "@types/redis": "^2.8.6",
     "jest": "^23.1.0",
+    "jest-each": "^23.1.0",
```

Comment:
I didn't know about 'jest-each', but that's a neat solution! it seems support for this has (very) recently become part of core Jest though, so maybe we don't even need the dependency (see [README](https://github.com/mattphillips/jest-each)).

Reply:
great catch!